docs/source/release_notes.rst

```diff
@@ -30,6 +31,7 @@ v1.20.0 Jan 5, 2023
         * Fix to eliminate fragmentation ``PerformanceWarning`` in ``feature_set_calculator.py`` (:pr:`2424`)
         * Fix serialization of ``NumberOfCommonWords`` feature with custom word_set (:pr:`2432`)
         * Improve edge case handling in NaturalLanguage primitives by standardizing delimiter regex (:pr:`2423`)
+        * Fix typo in ``_handle_binary_comparison`` function name and update ``set_feature_names`` docstring (:pr:`2388`)
```

Comment:
duplicate of line 28

Reply:
yup was just fixing this, idk what happened with my merge lol